documentation/bolt_installing_modules.md

```diff
@@ -14,17 +14,25 @@ are managed in Bolt projects. To find out how to opt in to the changes, see
 
 If you want to get started with a new Bolt project and you need specific modules
 from the Puppet Forge, you can install the modules and their dependencies with
-the `bolt project init` command.
+a Bolt command.
```

Comment:
```suggestion
using the Bolt command line.
```
I think "a Bolt command" is a bit too vague in this context - the reader immediately asks "Which command?"